config/jobs/kubernetes/sig-scalability/sig-scalability-golang.yaml

```diff
@@ -0,0 +1,25 @@
+periodics:
+- name: ci-build-and-push-k8s-at-golang-tip
+  interval: 8h
+  labels:
+    preset-service-account: "true"
+    preset-dind-enabled: "true"
+  annotations:
+    testgrid-dashboards: sig-scalability-experiments
```

Comment:
Let's have a separate dashboard for this: sig-scalability-golang
In addition to changing this line you'll have to do what I tried in https://github.com/kubernetes/test-infra/pull/17413

Reply:
Done.